src/app/snapp_test_transaction/snapp_test_transaction_lib.opam

```diff
@@ -0,0 +1,6 @@
+opam-version: "1.2"
```

Comment:
I would suggest naming the directory just `lib`.

I always feel like I'm typing too much for `archive/archive_lib`, would likely feel the same for this one.